src/main/java/com/stripe/param/AccountCreateParams.java

```diff
@@ -0,0 +1,2642 @@
+      }
+      this.requestedCapabilities.addAll(elements);
+      return this;
+    }
+
+    /**
+     * Add an element to `expand` list. A list is initialized for the first `add/addAll` call, and
+     * subsequent calls adds additional elements to the original list. See {@link
+     * AccountCreateParams#expand} for the field documentation.
+     */
+    public Builder addExpand(String element) {
+      if (this.expand == null) {
+        this.expand = new ArrayList<>();
+      }
+      this.expand.add(element);
+      return this;
+    }
+
+    /**
+     * Add an element to `requestedCapabilities` list. A list is initialized for the first
+     * `add/addAll` call, and subsequent calls adds additional elements to the original list. See
+     * {@link AccountCreateParams#requestedCapabilities} for the field documentation.
+     */
+    public Builder addRequestedCapability(RequestedCapability element) {
+      if (this.requestedCapabilities == null) {
+        this.requestedCapabilities = new ArrayList<>();
+      }
+      this.requestedCapabilities.add(element);
+      return this;
+    }
+
+    /**
+     * Add all map key/value pairs to `metadata` map. A map is initialized for the first
+     * `put/putAll` call, and subsequent calls add additional key/value pairs to the original map.
+     * See {@link AccountCreateParams#metadata} for the field documentation.
+     */
+    public Builder putAllMetadata(Map<String, String> map) {
+      if (this.metadata == null) {
+        this.metadata = new HashMap<>();
+      }
+      this.metadata.putAll(map);
+      return this;
+    }
+
+    /**
+     * Add a key/value pair to `metadata` map. A map is initialized for the first `put/putAll` call,
+     * and subsequent calls add additional key/value pairs to the original map. See {@link
+     * AccountCreateParams#metadata} for the field documentation.
+     */
+    public Builder putMetadata(String key, String value) {
+      if (this.metadata == null) {
+        this.metadata = new HashMap<>();
+      }
+      this.metadata.put(key, value);
+      return this;
+    }
+
+    /**
+     * An [account token](https://stripe.com/docs/api#create_account_token), used to securely
+     * provide details to the account.
+     */
+    public Builder setAccountToken(String accountToken) {
+      this.accountToken = accountToken;
+      return this;
+    }
+
+    /**
+     * Either a token, like the ones returned by [Stripe.js](https://stripe.com/docs/stripe.js), or
+     * a dictionary containing a user's bank account details.
+     */
+    public Builder setBankAccount(String bankAccount) {
+      this.bankAccount = bankAccount;
+      return this;
+    }
+
+    /**
+     * Either a token, like the ones returned by [Stripe.js](https://stripe.com/docs/stripe.js), or
+     * a dictionary containing a user's bank account details.
+     */
+    public Builder setBankAccount(Map<String, Object> bankAccount) {
```

Comment:
@remi-stripe when a field is polymorphic, the builder will have explicit setters for each type. So using the builder, one will have this effect of `setBankAccount` showing explicit options for all the possible types.
![image](https://user-images.githubusercontent.com/42550941/53468942-025cc500-3a11-11e9-8573-74d2b5ba9ffa.png)

The getter of the built object though will have return type Object that users have to cast, but that's not the expected use case.

Reply:
Ack okay. I got confused because I expected the String or Class of params for bank accounts like @ob-stripe did in .net with AnyOf